src/Silk.Core.Logic/Commands/Bot/AboutCommand.cs

```diff
@@ -5,30 +5,34 @@
 using DSharpPlus.CommandsNext;
 using DSharpPlus.CommandsNext.Attributes;
 using DSharpPlus.Entities;
+using Silk.Core.Discord;
 using Silk.Core.Discord.Utilities.HelpFormatter;
 using Silk.Extensions;
 
-namespace Silk.Core.Discord.Commands.Bot
+namespace Silk.Core.Logic.Commands.Bot
 {
     [Category(Categories.Bot)]
     public class AboutCommand : BaseCommandModule
     {
+        private readonly Main _main;
+        public AboutCommand(Main main) => _main = main;
+
         [Command("about")]
         [Description("Shows relevant information, data and links about Silk!")]
         public async Task SendBotInfo(CommandContext ctx)
         {
             var app = await ctx.Client.GetCurrentApplicationAsync();
             var dsp = typeof(DiscordClient).Assembly.GetName().Version;
 
-            int guilds = Discord.Bot.Instance!.Client.ShardClients.Values.SelectMany(x => x.Guilds).Count();
+            int guilds = _main.ShardClient.ShardClients.Values.SelectMany(x => x.Guilds).Count();
 
             var embed = new DiscordEmbedBuilder()
                 .WithTitle("About Silk!")
                 .WithColor(DiscordColor.Gold)
                 .AddField("Total guilds", $"{guilds}", true)
                 .AddField("Host", Environment.MachineName, true)
                 .AddField("Owner(s)", app.Owners.Select(x => x.Username).Join(", "), true)
-                .AddField("Bot version", Program.Version, true)
+                .AddField("Bot version", Shared.Constants.StringConstants.Version, true)
```

Comment:
Using `Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion` would allow you to set version inside package / assembly info, all whilst being more flexible than the current `X.X.X.X` versionning on packages.